spec/sidekiq/metrics/queues_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'influxdb'
+require 'sidekiq'
+
+RSpec.describe Sidekiq::Metrics::Queues do
+
+  let(:influxdb_client) { instance_double(InfluxDB::Client, config: config) }
+  let(:config) { instance_double(InfluxDB::Config, time_precision: 's') }
+  let(:sidekiq_queues) { instance_double(::Sidekiq::Stats::Queues, lengths: {"default" => 1}) }
```

Comment:
Will it work without the leading `::`? I hate these.

Reply:
Yes